resources/js/Pages/team/_Show.vue

```diff
@@ -10,7 +10,31 @@
     </div>
     <div class="md:flex mt-4">
         <div class="md:w-48 w-full flex-shrink-0 mr-2 text-center mb-2">
-            <img :src="team.image" class="inline" />
+            <img class="" :src="team.image || '/img/placeholder.png'" />
```

Comment:
You should commit the file `/img/placeholder.png` to the git repository as well :-)